src/ioctls/system.rs

```diff
@@ -374,6 +376,46 @@ impl Kvm {
         }
     }
 
+    /// AArch64 specific function to create a VM fd using the KVM fd with flexible IPA size.
```

Comment:
Should we also specify that `Cap::ArmVmIPASize` should be checked before calling this function?

Reply:
Yes. I have added this part. Thanks for reminding.